From bf11101a4c1efaf5b0d80168d5eedc0ae12d6e52 Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Thu, 6 Jun 2019 12:14:21 +0530
Subject: [PATCH 191/767] octeontx2: Fix mbox driver compilation dependency.

commit 86b36b335b7dc4f31c6b97aba9561daa4bca3f9f from
git@git.assembla.com:cavium/WindRiver.linux.git

Mbox driver needs to get compiled independently of whether AF/PF
driver enabled/disabled. This patch fixes that dependency.

Change-Id: I98445130cd1120e2ba3ab017eb90f74398a80249
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/Makefile b/drivers/net/ethernet/marvell/octeontx2/Makefile
index 4e7d6117b78d..563dc9cd6b36 100644
--- a/drivers/net/ethernet/marvell/octeontx2/Makefile
+++ b/drivers/net/ethernet/marvell/octeontx2/Makefile
@@ -3,5 +3,6 @@
 # Makefile for Marvell OcteonTX2 device drivers.
 #
 
+obj-$(CONFIG_OCTEONTX2_MBOX) += af/
 obj-$(CONFIG_OCTEONTX2_AF) += af/
 obj-y			   += nic/
-- 
2.31.1

